File: 657-robot-return-to-origin/robot-return-to-origin.c
bool judgeCircle(char* moves) {
    int r=0,d=0;
        for (int i = 0; moves[i] != '\0'; i++){
            if (moves[i]=='L'){
                r++;
            }
            else if (moves[i]=='R')
                r--;
            else if (moves[i]=='U')
                d++;
            else if (moves[i]=='D')
                d--;
        }
        return (r==0 && d==0);
}

File: 657-robot-return-to-origin/robot-return-to-origin.py
class Solution(object):
    def judgeCircle(self, moves):
        l=r=u=d=0
        for i in range(len(moves)):
            if moves[i]=='L':
                l+=1
            elif moves[i]=='R':
                r+=1
            elif moves[i]=='U':
                u+=1
            elif moves[i]=='D':
                d+=1
        return l == r and u == d
        